src/classroom.rb

```diff
@@ -0,0 +1,9 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+  end
+
+  has_many :students
```

Comment:
- [ ] The has-many / belongs-to relationships between Classroom and Student should be implemented according to this [article](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships`). Kindly take a careful look at the requirements.